packages/amplify-graphql-relational-transformer/package.json

```diff
@@ -34,6 +34,7 @@
     "@aws-cdk/aws-appsync": "~1.124.0",
     "@aws-cdk/aws-dynamodb": "~1.124.0",
     "@aws-cdk/core": "~1.124.0",
+    "amplify-cli-core": "^2.9.0",
```

Comment:
If the `amplify-cli-core` is dependency for feature flag, I suggest to move it to `peerDependencies`

Reply:
the newest feature flag is not yet release in CLI, after discussing with @alharris-at we'll go with this one and then have a separate PR to switch it after is released in CLI

Reply:
the `amplify-cli-core` is not needed anymore, letover after some refactor, removed, thank you